pom.xml

```diff
@@ -822,53 +822,6 @@ under the License.
 	</dependencyManagement>
 
 	<profiles>
-		<profile>
-			<id>scala-2.11</id>
-			<properties>
-				<scala.version>2.11.12</scala.version>
-				<scala.binary.version>2.11</scala.binary.version>
-			</properties>
-			<activation>
-				<property>
-					<name>!scala-2.12</name>
-				</property>
-			</activation>
-			<!-- Scala Shell doesn't currently work with Scala 2.12 so only include
-			when building for Scala 2.11. -->
-			<modules>
-				<module>flink-scala-shell</module>
-			</modules>
```

Comment:
I noticed that `flink-scala-shell` does not support Scala 2.12 yet. I just removed the conditional compilation, the module remains. It looks like https://github.com/apache/flink/pull/11895 addresses this but is still in-progress.